Lesson-4.done

AlexeyTretyakov98 opened the pull request.

AlexeyTretyakov98 committed:
Lesson-4.done

Marinarium commented on index.html:
```diff
@@ -1,10 +1,73 @@
+<div class="advantages">
+    <h2 class="heading">What do we offer</h2>
+    <hr class="highlight">
+<!--Var 2-->
```
this is a really bad practice in your comment. it will be impossible to adaptive for different devices without crutches
it's better to create an html structure according to components
![image](https://user-images.githubusercontent.com/25488411/156513688-56bdd416-10ac-479f-84f2-8efd253f182b.png)

Marinarium commented on index.html:
```diff
@@ -1,10 +1,73 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ru">
```
en)

Marinarium commented on index.html:
```diff
@@ -1,10 +1,73 @@
+    <!--      </div>-->
+    <!--    </div>-->
+    <!---->
+    <div class="content">
```
**Semantic recommendation** 
It's better to use `<ul>`  with` <li>` here
This is an obvious list of elements

Marinarium commented on index.html:
```diff
@@ -1,10 +1,73 @@
 </head>
 <body>
-<h1>lesson 1</h1>
+<div class="advantages">
```
**Semantic recommendation** 
It's better to use `<section> ` here.
The` <section>` tag defines a section in a document.
![image](https://user-images.githubusercontent.com/25488411/156405622-de79942f-de2a-4d42-8651-6de5a089d580.png)

Marinarium commented on index.html:
```diff
@@ -1,10 +1,73 @@
+    <div class="content">
+
+        <div class="vectors">
+            <div class="down">
```
Don't add wrappers if you can escape them
for example you could use only `<img> `here
```
img {
    width: 100px;
    height: 100px;
   object-fit: contain;
}
```
[Read more](https://www.w3schools.com/css/css3_object-fit.asp)

Marinarium commented on index.html:
```diff
@@ -1,10 +1,73 @@
+            <div class="down">
+                <img class="vectors-img" src="./img/headphones.png" alt="Support 24/7">
+            </div>
+            <div class="img-text">
```
Why do you need a wrapper for this text? Please, don't add wrappers for wrappers, if you can. This is an excess code

Marinarium commented on styles/style.css:
```diff
@@ -0,0 +1,87 @@
+/*    line-height: 28px;*/
+/*    text-align: center;*/
+/*    color: #383838;*/
+/*}*/
```
try to avoid comments in your last version of code, delete)

Marinarium commented on styles/style.css:
```diff
@@ -0,0 +1,87 @@
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+}
+
```
empty row, please, use one style of code writing

Marinarium commented on styles/style.css:
```diff
@@ -0,0 +1,87 @@
+    font-style: normal;
+    font-weight: normal;
+    font-size: 17px;
+    line-height: 28px;
```
**Just a recommendation**
it's better to use `line-height` with values **without px**
examples
```
line-height: 2.5;
line-height: 3em;
line-height: 150%;
```

AlexeyTretyakov98 committed:
Lesson-4.
add <section>
add property for img
del additional div
del comments
del empty row
removed px for line-height

AlexeyTretyakov98 committed:
Lesson-4.
add <section>

AlexeyTretyakov98 committed:
Lesson-4.

AlexeyTretyakov98 committed:
Lesson-4.

AlexeyTretyakov98 committed:
refactor according to code review

AlexeyTretyakov98 committed:
resolve conflicts
